frontend/packages/pipelines-plugin/src/types/coreTekton.ts

```diff
@@ -11,3 +11,14 @@ export type TektonResource = {
   optional?: boolean;
   type: string; // TODO: limit to known strings
 };
+
+export type TektonResultsDefinition = {
```

Comment:
Let's delete this type and add it directly to the `Pipeline` and `Task`. 

- Pipeline needs all three `name`, `description?` and `value`
- Task needs just two `name` and `description?`

Reply:
updated